types/index.d.ts

```diff
@@ -633,6 +633,14 @@ declare namespace jest {
      */
     toBeWithin(start: number, end: number): any;
 
+    /**
+     * Use `.toBeWithinRange` when checking if an array has elements in range min (inclusive) and max (inclusive).
+     *
+     * @param min
+     * @param max
+     */
+    toBeWithinRange(min: number, max: number): any;
```

Comment:
Please rebase to replace the `any`. Also add a signature to the `Matchers` interface.

Reply:
Done